vendor/github.com/containers/libpod/pkg/apparmor/apparmor_linux_template.go

```diff
@@ -24,8 +23,6 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
   deny @{PROC}/sys/[^k]** w,  # deny /proc/sys except /proc/sys/k* (effectively /proc/sys/kernel)
   deny @{PROC}/sys/kernel/{?,??,[^s][^h][^m]**} w,  # deny everything except shm* in /proc/sys/kernel/
   deny @{PROC}/sysrq-trigger rwklx,
-  deny @{PROC}/mem rwklx,
-  deny @{PROC}/kmem rwklx,
```

Comment:
Do we know where these went?  Looks like libpod's template landed without them in containers/libpod#1063.

Reply:
libpod's template comes from Moby/Docker which does not set those two rules. I'd prefer to remain compatible with Docker as much as possible to avoid any hiccups with workloads that run on Docker but not on Podman and CRI-O.